### PR TITLE
Initial spec

### PR DESCRIPTION
**Pull request by kkohbrok**

### CODE REVIEW

File: src/spec/delivery_service.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Delivery service (DS)
```
Comment (raphaelrobert) [inline src/spec/delivery_service.md:1]:
```suggestion
# Delivery Service (DS)
```

File: src/spec/authentication_service/evolving_identities.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Any commit to such a change needs to be sent to the AS for publication, but also to all of the user's groups. In order for all these parties to be able to track changes to the user's identity, the commit needs to be in an MLS plaintext message. Any recipient of such a commit can validate the visible information in the commit, and in particular verify the signature against the current state of the EID.
+
+### Cross-signing
+
+One goal of the EID is to implement two-directional client cross signing, i.e. have both the adding client and the added client confirm the addition with their signatures. To achieve this, recipients of a commit to the EID that contains a client addition MUST wait for a second commit sent by the new client that only contains an update by that client.
+
+TODO: Add link to explanation of how the DS manages groups.
+
+## Future work: User ID changes
```
Comment (raphaelrobert) [inline src/spec/authentication_service/evolving_identities.md:39]: The spelling of "ID" is inconsistent throughout the document: ID/id. I think it should be ID throughout.

**Review by raphaelrobert — CHANGES_REQUESTED** [review]
Thanks, just some minor stylistic remarks and suggestions. The content is WIP anyway but already looks good.

**Review by kkohbrok — COMMENTED** [review]
Thanks for the review! I'll address the id/ID inconsistency and make a pass over the TODOs before merging.

**Pull request merged**